docs/en/changes/changes.md

```diff
@@ -21,6 +21,10 @@
 * Remove legacy OAL `percentile` functions, `p99`, `p95`, `p90`, `p75`, `p50` func(s).
 * Revert [#8066](https://github.com/apache/skywalking/pull/8066). Keep all metrics persistent even it is default value.
 * Skip loading UI templates if folder is empty or doesn't exist.
+* Support the `NETWORK` type of eBPF Profiling task.
+* Support `sumHistogram` in `MAL`.
+* [Breaking Change] Let the eBPF Profiling task support to the service instance level, please delete
+    index/table: `ebpf_profiling_task`.
```

Comment:
Is this `breaking change` description precisely? 
Do you mean we only supported tag based profiling before

Reply:
Yes. We only supported tag-based profiling before, now we add the instance based. Also, we change the `ID` getting strategy. Now we use the logical ID instead of the previous data ID.

Reply:
```suggestion
* [Breaking Change] Make the eBPF Profiling task support to the service instance level, index/table `ebpf_profiling_task` is required to be re-created when bump up from previous releases.
```

Reply:
> we change the ID getting strategy.

About this, I think the breaking is only relative to the previous data, or really affects the index/alias/table schema?

Reply:
No, it doesn't affect the schema, I just add a new field as a logical ID. Generating and getting logical IDs is the same as normal fields.

Reply:
If we don't rebuild the table, an exception will be thrown when querying the UI, because we need to use the logical ID in the query, which does not exist in the old version. Unless adding a judgment to verify whether the logical ID exists, if it does not exist, then Use primary keys. ES storage is ok, but MySQL doesn't work and I'm not sure the banyanDB works or not.